index.html

```diff
@@ -0,0 +1,676 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css">
+</head>
+
+<body>
+
+
+    <main>
+        <div class="blok" id="header">
+
+
+
+            <div class="nav" style="height:6rem; width:100vw">
+
+                <h2 id="logo" style="position:relative;left:2rem;top:1.5rem">U.I. Ninja</h2>
+
+
+
+
+                <ul>
+                    <li style="position:relative;left:-2rem;top:0rem;"><a href="home.html">Home</a> </li>
+                    <li style="position:relative;left:-2rem;top:0rem"><a href="#">Docs</a></li>
+                    <li style="position:relative;left:-2.3rem;top:0rem"><a href="https://github.com/Ayush-kumar1/Component-Library" target="_blank">Github</a></li>
+                </ul>
+
+            </div>
+        </div>
+
+
+
+
+        <!-- SIDEBAR  -->
+
+        <div id="page-wrap">
+            <div class="blok" id="sidebar">
+
+                <!-- <br><br> -->
+
+                <h3>Components</h3>
+
+                <ul>
+                    <!-- <li><a href="#header">Header</a></li> -->
```

Comment:
Why does clicking on sidebar links take me to another another page, shouldn't they scroll to the section in the doc page itself?